src/components/PokemonCard/PokemonCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { Pokemon } from 'services/pokemon/pokemon.types';
+
+interface Props {
+  pokemon: Pokemon;
+}
+
+const PokemonCard = ({ pokemon }: Props) => {
+  const { name, sprites } = pokemon;
```

Comment:
you can make this inside of props declaration (row 7), no point to make it inside the component